docs/src/content/docs/fr/guides/i18n.mdx

```diff
@@ -271,3 +273,16 @@ export const collections = {
 ```
 
 Consultez [« Définir un schéma de collection de contenus »](https://docs.astro.build/fr/guides/content-collections/#defining-a-collection-schema) dans la documentation d'Astro pour en savoir plus sur les schémas de collection de contenus.
+
+## Accès à la langue locale actuelle
```

Comment:
```suggestion
## Accès aux paramètres régionaux
```

[Reference](https://fr.wikipedia.org/wiki/Param%C3%A8tres_r%C3%A9gionaux): looks like this is the proper word for "locales" which is an anglicism in French.